Marlin/src/pins/stm32f1/pins_CREALITY_CR4NS.h

```diff
@@ -76,31 +76,34 @@
 //
 #define HEATER_BED_PIN                      PB2   // HOT BED
 #define FAN1_PIN                            PC1   // extruder fan
-//#define FAN2_PIN                          PB1   // Controller fan FET
 
 //
-// Auto fans
+// Steppers
 //
-//#ifndef CONTROLLER_FAN_PIN
-//  #define CONTROLLER_FAN_PIN          FAN2_PIN
-//#endif
-
 #if HAS_TMC_UART
   // Reduce baud rate to improve software serial reliability
   #define TMC_BAUD_RATE 19200
 
   // Software serial
   #define X_SERIAL_TX_PIN                   PB12
   #define X_SERIAL_RX_PIN        X_SERIAL_TX_PIN
+  #define X_DIAG_PIN                        PB10
```

Comment:
Dedicated DIAG pins are interesting. Is this board pre-built with Trinamic drivers with UART, or does it have other types of stepper drivers? A board that has dedicated DIAG pins may also make these pins available for Trinamic SPI and standalone drivers.